server/src/com/cloud/network/router/VpcVirtualNetworkApplianceManagerImpl.java

```diff
@@ -685,6 +685,16 @@ public void finalizeStop(final VirtualMachineProfile profile, final Answer answe
     }
 
     @Override
+    public boolean start() {
+        return true;
```

Comment:
@harikrishna-patnala Hi Hari, can you explain what the changes are doing to fix the issue of multiple thread synchronisation ? Also, what happens across multiple mgmt servers since we don't have any distributed consensus implementation in place, I suppose the statcollector values might be different.